src/BodyFunctionality/BodyPart.java

```diff
@@ -0,0 +1,487 @@
+package BodyFunctionality;
+
+import javafx.scene.paint.Color;
+
+import java.util.ArrayList;
+
+public class BodyPart
```

Comment:
Needs to implement BodyPartInterface.
None of the classes that inherit this need to implement BodyPartInterface
(It's implemented through inheritance)

Reply:
So, you are suggesting to have BodyPart implement the BodyPartInterface, not the parts that inherit from BodyPart?

Reply:
Yee